build-logic/jvm/src/main/kotlin/gradlebuild.unittest-and-compile.gradle.kts

```diff
@@ -242,7 +242,7 @@ fun configureTests() {
         if (BuildEnvironment.isCiServer) {
             configureRerun()
             retry {
-                maxRetries.convention(1)
+                maxRetries.convention(2)
```

Comment:
Is this configuration for performance tests as well? Then we probably should reduce it to 1 there. Same probably for smoke tests. WDYT?

Reply:
`PerformanceTest` is not visible in this scope, so I did the judgement by project name.